etl_manager/etl.py

```diff
@@ -351,45 +360,34 @@ def run_job(self, sync_to_s3_before_run = True) :
         self._job_run_id = response['JobRunId']
 
     @property
-    def job_status(self) :
-        if self.job_name is None or self.job_run_id is None :
-            response = "Glue Job Obj missing job_name or job_run_id"
-        else :
-            try :
-                response = _glue_client.get_job_run(JobName = self.job_name, RunId = self.job_run_id)
-            except :
-                response = 'No glue job with job_name: {} and job_run_id: {}'.format(self.job_name, self.job_run_id)
-        return response
+    def job_status(self):
+        if self.job_run_id is None:
+            raise JobNotStarted('Missing "job_run_id", have you started the job?')
+
+        if self.job_name is None:
```

Comment:
this is a true exception but could JobNotStarted just be returned as a status itself?

Reply:
I see what you mean. 

Probably not in this case because this is an instance method and what the client code is really asking for is the status of the specific instance ("run") of the Glue Job. 
The problem is that if the user didn't start the job there is not `run_id` so asking for the state doesn't make much sense.

But maybe I didn't understand your point, apologies if that's the case.